crates/cairo-profiler/tests/contracts/builtins_simple/Scarb.toml

```diff
@@ -15,3 +15,6 @@ sierra = true
 
 [tool.scarb]
 allow-prebuilt-plugins = ["snforge_std"]
+
+[profile.dev.cairo]
```

Comment:
Added this, because I also wanted to make sure casm sizes are generated properly for inlined functions - this is also the reason why the artifacts changed a bit.